src/main/java/dev/jbang/dependencies/DependencyResolver.java

```diff
@@ -49,7 +49,8 @@ public DependencyResolver addArtifacts(List<ArtifactInfo> artifacts) {
 	}
 
 	public DependencyResolver addClassPath(String classPath) {
-		return addArtifact(DependencyCache.findArtifactByPath(new File(classPath)));
+		// WARN need File here because it's more lenient about paths than Path!
```

Comment:
interesting - got example where Path fails but File is okey?

Reply:
Yes, the `Class-Path` element of the MANIFEST uses an URI-like format for paths which, on WIndows, turns `C:\foo\bar` into `/C:/foo/bar` then when you try to read it again FIle will happily parse it while Path complains that `:` is not a valid char. Perhaps this should be handled in some other way, but for now I decided just to keep using FIle in this case.